cpan/ExtUtils-PL2Bat/t/make_executable.t

```diff
@@ -31,3 +32,29 @@ push @files, grep { -f } map { $filename.$_ } split / $Config{path_sep} /x, $ENV
 is scalar(@files), 1, "Executable file exists";
 
 unlink $filename, @files;
+
+# the test needs CWD in PATH to check the created .bat files, but under win2k
+# PATH must not be too long. so to keep any win2k smokers happy, we construct
+# a new PATH that contains the dirs which hold cmd.exe, perl.exe, and CWD
+
+sub construct_test_PATH {
+	my $perl_path = $^X;
+	my $cmd_path = $ENV{ComSpec} ||  `where cmd`; # where doesn't seem to work on all windows versions
```

Comment:
should not we check for `$?` when it fails?

Reply:
Sensible too.